public/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  Name: <input type = 'text' id = 'a1'/>
+  Comment: <input type = 'text' id = 'a2'/>
+  <input type = 'submit' id = 'b1'>
+<script>
+  const button = document.getElementById('b1');
+  button.onclick = () => {
+    const name = document.getElementById('a1').value;
+    const comment = document.getElementById('a2').value;
+    const xhttp = new XMLHttpRequest();
+    xhttp.open('GET', `/XMLHttpRequest?a1=${name}&a2=${comment}`);
```

Comment:
use better variable names in the url as well

Reply:
Yes, I agree 😃